spec/core/basicobject/fixtures/common.rb

```diff
@@ -0,0 +1,11 @@
+module BasicObjectSpecs
+  class BOSubclass < BasicObject
+    def self.kernel_defined?
+      defined?(Kernel)
+    end
+
+    # NATFIXME: This results in "wrong number of arguments (given 0, expected 1+) (ArgumentError)"
+    # include ::Kernel
```

Comment:
This is most likely a parser bug

Reply:
This fixes it at well:
```ruby
include Kernel
```
So yes, parser error is very likely here.

Reply:
Fixed in natalie-lang/natalie_parser#53